packages/swapper/src/swappers/OneInchSwapper/getTradeQuote/getTradeQuote.ts

```diff
@@ -74,7 +74,12 @@ export async function getTradeQuote(
     affiliateBps,
   )
 
-  const rate = getRate(quote).toString()
+  const rate = getRate({
```

Comment:
See below - `getRate` moved to root swapper utils and generalized vs. taking a OneInch specific response - actual logic doesn't change.